### DISCUSSION TITLE
DOTENV extension only works with webpack

### DISCUSSION THREAD

**Discussion by murilolivorato** [quasarframework/quasar#16287] [CLI - SSR mode] [answered]
hello  , I used to install dot env like this in  quasar project  like this -
```
quasar ext add @quasar/dotenv
```

I created a nwe project with VITE  + Quasar . and  I installed at the same way .
but , when I gave the command -
```
quasar dev -m ssr  

it shoes this error
 App • ⚠️  Extension(@quasar/dotenv): Dependency not found - @quasar/app. Please install it.

```

I saw the documentations  and it is telling to I install like this now -

```
npm install dotenv --save
```

and change this config -
```
build: {
  env: require('dotenv').config().parsed
}
```

but didnt work ,  It showld explain better how to install it .
I  removed this project with VITE , and installed with webpack AND installed dot env like this -
```
quasar ext add @quasar/dotenv
```
and it worked , BUT with VITE does not works .

**Comment by metalsadman** [comment, marked as answer]
you don't need to use dotenv extension or dotenv library in vite, it's already baked in https://vitejs.dev/guide/env-and-mode.html#env-files

**Comment by murilolivorato** [comment]
It worked , thanks
